types/react-animated-modal/index.d.ts

```diff
@@ -0,0 +1,58 @@
+// Type definitions for react-animated-modal 1.1
+// Project: https://github.com/morfsys/react-animated-modal
+// Definitions by: Manish Kumar <https://github.com/manishprivet>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+export type AnimationTypes =
+    | "pulse"
+    | "rubberBand"
+    | "shake"
+    | "swing"
+    | "tada"
+    | "wobble"
+    | "jello"
+    | "bounceIn"
+    | "bounceInDown"
+    | "bounceInLeft"
+    | "bounceInRight"
+    | "bounceInUp"
+    | "fadeIn"
+    | "fadeInDown"
+    | "fadeInLeft"
+    | "fadeInRight"
+    | "fadeInUp"
+    | "flip"
+    | "flipInX"
+    | "flipInY"
+    | "lightSpeedIn"
+    | "rotateIn"
+    | "rotateInDownLeft"
+    | "rotateInDownRight"
+    | "rotateInUpLeft"
+    | "rotateInUpRight"
+    | "slideInUp"
+    | "slideInDown"
+    | "slideInLeft"
+    | "slideInRight"
+    | "zoomIn"
+    | "zoomInDown"
+    | "zoomInLeft"
+    | "zoomInRight"
+    | "zoomInUp"
+    | "hinge"
+    | "jackInTheBox"
+    | "rollIn"
+    | "bounce"
+    | "flash";
+
+export interface ModalProps {
+    closemodal: () => any;
+    visible: boolean;
+    type: AnimationTypes;
```

Comment:
```suggestion
    /** @default 'flipInX' */
    type?: AnimationTypes | undefined;
```
this is not reuqired, it takes default implemmentatio (not documented though):
https://github.com/morfsys/react-animated-modal/blob/master/src/index.js#L94

Reply:
Yup, thanks for the pick